src/Tmds.Ssh/SftpClient.cs

```diff
@@ -378,6 +378,12 @@ public async ValueTask DownloadFileAsync(string remoteFilePath, string localFile
         await channel.DownloadFileAsync(remoteFilePath, localFilePath, overwrite, cancellationToken);
     }
 
+    public async ValueTask DownloadFileAsync(string remoteFilePath, Stream destination, CancellationToken cancellationToken = default)
+    {
+        var channel = await GetChannelAsync(cancellationToken).ConfigureAwait(false);
+        await channel.DownloadFileAsync(remoteFilePath, destination, null, null, true, cancellationToken);
```

Comment:
Please remove the constant arguments (`null, null, true`) from the SftpChannel method being called here.